src/apis/vc.ts

```
@@ -0,0 +1,90 @@
+
+import { apiRequest } from '@/apis/apiClient';
+
+const APIURL_VC = {
+  GET_VC_LIST: (projectSeq: number) => `/vc/${projectSeq}`,
+  CREATE_VC: (srcSeq: number, trgSeq: number) =>
+    `/vc/result?srcSeq=${srcSeq}&trgSeq=${trgSeq}`,
+  REMOVE_SRC: `/vc/src`,
+};
+
+interface IVCProps<T = {}> {
+  projectSeq: number;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)